### PR TITLE
Add `tolerations` to `Placement` for GRC

### PR DESCRIPTION
**Pull request by dhaiducek**
Policy and PolicySets should not respond to unreachable clusters, so we need to add tolerations to Placement by default. Tolerations didn't exist previously, so I've added them to `PlacementSection` through a `defaultPlacementSpec`.

In addition:
- Removes references to "ManagedClusterSetBindings" (since this is a kind the console user doesn't need to know about on the GRC page)
- Updates tooltip wording to be more accurate
- Surfaces a previously commented out "description" for Placement

ref: https://issues.redhat.com/browse/ACM-10200

### CODE REVIEW

File: frontend/src/wizards/Placement/PlacementSection.tsx [hunk 1]
```diff
@@ -240,7 +248,7 @@ export function PlacementSection(props: {
               clusters={props.clusters}
               createClusterSetCallback={props.createClusterSetCallback}
               alertTitle={t(
-                'ClusterSets failed to load. Check the ManagedClusterSetBinding resource to verify your selected namespace.'
+                'ClusterSets failed to load. Verify that there are ClusterSets bound to your selected namespace.'
```
Comment (yiraeChristineKim) [inline frontend/src/wizards/Placement/PlacementSection.tsx:251]: Should translate to diff langs?

Comment (yiraeChristineKim) [reply frontend/src/wizards/Placement/PlacementSection.tsx:251]:
```
"i18n:fix:frontend": "cd frontend && npm run i18n:fix --",
```

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by dhaiducek** [comment]
Please take another look. I've made some updates:
- Changed the `tolerations` property to `defaultPlacementSpec` to provide additional flexibility for the future
- Enabled and edited a previously commented-out "description" (to be reviewed also by the Doc team)
- Updated some tooltip wording (the ClusterSet tooltip had been copied from Label expressions so was inaccurate)
- Moved the ClusterSet alert up for visibility and in an attempt to decouple it from the ClusterSets field.

File: frontend/src/wizards/Placement/PlacementSection.tsx [hunk 2]
```diff
@@ -203,7 +210,7 @@ export function PlacementSection(props: {
   return (
     <Section
       label={t('Placement')}
-      // description="Placement selects clusters from the cluster sets which have bindings to the resource namespace."
+      description="Placement selects clusters from the cluster sets which have been bound to the resource namespace. An empty Placement returns all available clusters from all bound cluster sets."
```
Comment (dockerymick) [inline frontend/src/wizards/Placement/PlacementSection.tsx:213]: Use Placement resources to select clusters from the cluster sets that you have bound to the resource namespace. An empty Placement returns all available clusters from all bound cluster sets.

File: frontend/public/locales/en/translation.json [hunk 3]
```diff
@@ -2303,9 +2302,9 @@
   "Select an inventory": "Select an inventory",
   "Select at least one cluster set to deploy application resources.": "Select at least one cluster set to deploy application resources.",
   "Select cluster label": "Select cluster label",
+  "Select cluster sets from which to select clusters. If no cluster sets are selected, all clusters will be selected from all cluster sets bound to the namespace.": "Select cluster sets from which to select clusters. If no cluster sets are selected, all clusters will be selected from all cluster sets bound to the namespace.",
```
Comment (dockerymick) [inline frontend/public/locales/en/translation.json:2305]:
```suggestion
  "Select cluster sets from which to select clusters. If you do not select a cluster set, all clusters are selected from all cluster sets bound to the namespace.": "Select cluster sets from which to select clusters. If you do not select a cluster set, all clusters are selected from all cluster sets bound to the namespace.",
```

**Review by dockerymick — CHANGES_REQUESTED** [review]
left some suggestions

**Comment by dhaiducek** [comment]
Thanks for the reviews, all! I think all relevant parties have reviewed this PR, so it should be good to re-review and merge.

**Review by mprahl — APPROVED** [review]

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by dhaiducek** [comment]
...and by "ready", I mean as soon as I fix the linting... 🤦‍♂️

**Comment by dhaiducek** [comment]
Obligatory screenshots:
image
image

**Review by dockerymick — APPROVED** [review]

**Pull request merged**